docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -38,36 +38,37 @@ Be aware that if you are running out of time during your rental, you can [extend
 Before configuring color detection, enable the rover's camera to get a better sense of what it perceives.
 
 - If you are running this tutorial with a [rented Viam Rover](https://app.viam.com/try), enable both provided cameras: the front-facing camera and the overhead cam.
-  In the `viam_base` component panel under the **Control** tab, enable both the `cam` for the front-facing camera and the `overhead-cam:overheadcam` for an overhead view of your rover.
+  In the `viam_base` component panel under the **CONTROL** tab, enable both the `cam` for the front-facing camera and the `overhead-cam:overheadcam` for an overhead view of your rover.
 
   ![The viam_base component panel showing both the 'cam' and 'overheadcam' camera feeds enabled.](get-started/try-viam/try-viam/enable-both-cameras.png)
 
   You can also view and control the camera streams from the [individual camera component panels](/get-started/try-viam/try-viam-tutorial/#camera-control).
 
 - If you are running this tutorial on [your own Viam Rover](/get-started/try-viam/rover-resources/), enable the front facing camera.
-  If you are using the `ViamRover` [fragment](/get-started/try-viam/rover-resources/rover-tutorial-fragments/) with your rover, the front facing camera is named `cam` and can be enabled in the `viam_base` component panel under the **Control** tab.
+  If you are using the `ViamRover` [fragment](/get-started/try-viam/rover-resources/rover-tutorial-fragments/) with your rover, the front facing camera is named `cam` and can be enabled in the `viam_base` component panel under the **CONTROL** tab.
 
 ## Add the vision service to detect a color
 
+{{% alert title="Tip" color="tip" %}}
+If you are using a rented Viam rover, you can skip this section.
+Your rover is already configured with a `#7a4f5c` color detector called `my_color_detector`.
+{{% /alert %}}
+
 This tutorial uses the color `#7a4f5c` or `rgb(122, 79, 92)` (a reddish color).
 
 **Hex color #7a4f5c**: {{<imgproc src="/tutorials/try-viam-color-detection/7a4f5c.png" resize="150x" declaredimensions=true alt="A color swatch for the color that you will be detecting with your color detector. It's a reddish, maroon color.">}}
 
-Navigate to your machine's **Config** tab on the [Viam app](https://app.viam.com/robots) and configure your [vision service color detector](/ml/vision/#detections):
+Navigate to your machine's **CONFIGURE** tab on the [Viam app](https://app.viam.com/robots) and configure your [vision service color detector](/ml/vision/#detections):
 
 {{< tabs >}}
 {{% tab name="Builder" %}}
 
-1. Click the **Services** subtab and click **Create service** in the lower-left corner.
-
-1. Select the `Vision` type, then select the `Color Detector` model.
-
+1. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
```

Comment:
I don't know who started this, but I quite like it, wdyt?
```suggestion
1. Click the + (Create) icon next to your machine part in the left-hand menu and select **Service**.
```

Reply:
haha that was me I'm just inconsistent 😄 happy to add. Would you want me to make it a ticket to add the button name to all the resource configuration instructions?